docs/csharp/whats-new/version-update-considerations.md

```diff
@@ -57,4 +57,4 @@ New code:
 public double CalculateSquare(in double value) => value * value;
 ```
 
-The [What's new](index.md) articles note if introducing a feature that affects public declarations is source compatible or binary compatible.
+The [What's new](csharp-8.md) articles note if introducing a feature that affects public declarations is source compatible or binary compatible.
```

Comment:
This one should stay as is.  The `index.md` file will always be redirected to the latest version (when I add csharp-9.md, for example). That way there's one location to update.

```suggestion
The [What's new](index.md) articles note if introducing a feature that affects public declarations is source compatible or binary compatible.
```

Reply:
Ah, makes sense. I should check for this in my other PRs. Currently my tool doesn't have a way of excluding redirects, but I'll know to look for this in the future (there's always some manual manipulation involved, e.g. to fix link text).